src/components/search-query.jsx

```diff
@@ -96,22 +108,48 @@ HitComponent.propTypes = {
     office_title: PropTypes.string,
   }),
 };
-HitComponent.defaultProps = {
-  hit: {},
+
+Hits.defaultProps = {
+  hits: {},
 };
 
+const Results = connectStateResults(({ searchState }) =>
+  (searchState && searchState.query ? (
+    <div>
+      <CustomHits />
+    </div>
+  ) : (
+    <div>No query</div>
+  )));
+
 const SearchQuery = () => (
   <InstantSearch searchClient={searchClient} indexName="election">
     <div className="grid">
       <div className="grid-col-2 election-checkboxes">
         <ClearRefinements clearsQuery="true" />
         <h3>Election Title</h3>
-        <RefinementList attribute="election_title" />
+        <RefinementList
+          attribute="election_title"
+          /* List refinements ordered by date descending */
+          transformItems={items =>
+            orderBy(items, (i) => {
+              /* extract the election date */
```

Comment:
I appreciate the comments here! Eventually we might want to look at moving these helper functions to a separate file, but I think that since this is small enough and encapsulated within `RefinementList`, it's okay to leave in-line for now.